components/modal/gallery/src/index.js

```diff
@@ -0,0 +1,126 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import SuiModal from '@schibstedspain/sui-modal-basic'
+import ImageSlider from '@schibstedspain/sui-image-slider'
+
+const NO_OP = () => {}
+
+const DEFAULT_COUNTER_TEXT_FORMATTER = (current, total) => `${current} / ${total}`
+
+class ModalGallery extends Component {
+  constructor (props) {
```

Comment:
For this kind of constructor, you could just do:

```
class ModalGallery extends Component {
  state = { currentSlide: props.initialSlide }
  ...
```

Reply:
OK 👍 
I will change this.